### PR TITLE
update-docker-deployment

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md [hunk 1]
```diff
@@ -89,9 +97,9 @@ nebula-docker-compose_storaged1_1   ./bin/nebula-storaged --fl ...   Up (healthy
 nebula-docker-compose_storaged2_1   ./bin/nebula-storaged --fl ...   Up (healthy)   0.0.0.0:32783->12000/tcp, 0.0.0.0:32781->12002/tcp, 0.0.0.0:32779->44500/tcp, 44501/tcp
 ```
 
-Nebula Graph provides services to the clients through port 3699 by default. You can adjust the port number by modifying the [network configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/3.configurations/3.meta-config/#networking_configurations).
+Nebula Graph provides services to the clients through port `3699` by default. You can adjust the port number by modifying the [network configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/3.configurations/3.meta-config/#networking_configurations).
```
Comment (whitewum) [inline docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md:100]: is it 3699 now?

File: docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md [hunk 2]
```diff
@@ -23,20 +23,20 @@ If you are reading this topic with the questions listed below, click them to jum
...
+* If you are deploying Nebula Graph as a non-root user, grant the user with Docker-related privileges. For a detailed instruction, see [Docker document: Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
 
 * You have started the Docker service on your host.
 
 * If you have already deployed another version of Nebula Graph with Docker Compose on your host, to avoid compatibility issues，back up [the service data](#check_the_service_data_and_logs) if you need, and delete the `nebula-docker-compose/data` directory.
 
 ## How to deploy
 
-1. Clone the `v2-preview` branch of the `nebula-docker-compose` repository to your host with Git.
+1. Clone the `master` branch of the `nebula-docker-compose` repository to your host with Git.
```
Comment (whitewum) [inline docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md:34]: errrr.

**Review by whitewum — COMMENTED** [review]
the master branch default port is not 3699 now

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md [hunk 3]
```diff
@@ -68,7 +68,15 @@ If you are reading this topic with the questions listed below, click them to jum
...
-4. Connect to Nebula Graph. For detailed instructions, see [Use nebula-console to connect to Nebula Graph](3.connect-to-nebula-graph.md/#use_nebula-console_to_connect_to_nebula_graph).
+4. Connect to Nebula Graph.
+
+    ```bash
+    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
+    ```
+
+    ```bash
+    docker> nebula-console -u <user> -p <password> --address=graphd --port=3699
+    ```
```
Comment (oldLady344) [inline docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md:79]: 4. Use Nebula Console to connect to Nebula Graph. OR "Get access to Nebula Graph with Nebula Console. "
 
```bash
    # Start a Nebula Console service in a Docker container? 
    $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
    
    # Connect Nebula Console to Nebula Graph 
    docker> nebula-console -u <user> -p <password> --address=graphd --port=3699
    ```

according to this thread: https://discuss.nebula-graph.com.cn/t/topic/2209/15, I think maybe some comments should be added to explain these commands.

**Pull request merged**